### PR TITLE
fix(textarea): updating disable label color

### PR DESCRIPTION
**Pull request by ckrook**
## **Describe pull-request**  
The label color for text area was incorrect and have in this PR been updated according to the [Figma design](https://www.figma.com/design/d8bTgEx7h694MSesi2CTLF/Tegel-UI-Library?node-id=34279-126066&node-type=frame&t=OMmW16UPWFXCIjYE-0).

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3621](https://tegel.atlassian.net/browse/CDEP-3621)    

## **How to test**  
**Test 1**
1. Go to the aws preview link
2. Navigate to the textarea component
3. Add an label outside the textarea and toggle disabled.
4. Now verify that the color is correct according to the [Figma Design System](https://www.figma.com/design/d8bTgEx7h694MSesi2CTLF/Tegel-UI-Library?node-id=34279-126066&node-type=frame&t=OMmW16UPWFXCIjYE-0)

**Test 2**
1. Add an label **inside** the textarea and toggle disabled.
2. Now verify that the color is correct according to the [Figma Design System](https://www.figma.com/design/d8bTgEx7h694MSesi2CTLF/Tegel-UI-Library?node-id=34279-126066&node-type=frame&t=OMmW16UPWFXCIjYE-0)

## **Checklist before submission**
- [ ] I have added unit tests for my changes (if applicable)
- [x] All existing tests pass
- [ ] I have updated the documentation (if applicable)
- [x] Not breaking production behavior
- [ ] Behavior available in storybook with documented descriptions (if applicable)
- [x] `npm run build-all` without errors

## **Suggested test steps**
- [x] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [x] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

## **Screenshots**  

outside-pr-fix
inside-pr-fix
outside-production
inside-produciton

[CDEP-3621]: https://tegel.atlassian.net/browse/CDEP-3621?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by theJohnnyMe — DISMISSED** [review]
Nice one!

**Comment by theJohnnyMe** [comment]
For anyone involved, value in Figma is off, it shows 700 instead of 800. Charlotte is aware and confirmed that we should go with 800. Ticket is created for designers to update Figma values.

File: packages/core/src/components/textarea/textarea-vars.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
   --tds-textarea-disabled-background: var(--tds-textarea-disabled-background-primary);
   --tds-textarea-disabled-placeholder: var(--tds-grey-400);
   --tds-textarea-disabled-label: var(--tds-grey-400);
+  --tds-textarea-disabled-label-inside-color: var(--tds-grey-700);
```
Comment (nathalielindqvist) [inline packages/core/src/components/textarea/textarea-vars.scss:15]: Regarding the discussion had earlier today, should this be changed to grey-800?

**Review by nathalielindqvist — COMMENTED** [review]
Should label text for Text Field in disabled state when in dark mode be updated to grey-800 as well? Since it's currently grey-700? :)

File: packages/core/src/components/textarea/textarea-vars.scss (hunk 1 above)
Comment (theJohnnyMe) [reply packages/core/src/components/textarea/textarea-vars.scss:15]: Possibly but we did not confirm that with Charlotte, only the outside one. 
Can you please double-check with her? 
Suck that Figma is not up-to-date and a source of truth.

Comment (nathalielindqvist) [reply packages/core/src/components/textarea/textarea-vars.scss:15]: It has now been verified with Charlotte that label inside should also be grey-800 when the component is disabled :)

**Pull request closed without merging**